src/midje/parsing/1_to_explicit_form/metaconstants.clj

```diff
@@ -4,7 +4,7 @@
   (:import midje.data.metaconstant.Metaconstant))
 
 (defn predefine-metaconstants-from-form [form]
-  (let [metaconstant-symbols (filter data/metaconstant-symbol? (tree-seq coll? seq form))]
+  (let [metaconstant-symbols (set (filter data/metaconstant-symbol? (tree-seq coll? seq form)))]
```

Comment:
I noticed that `(intern *ns* symbol (Metaconstant. symbol {} nil))` was being run multiple times for the same symbol. Figured it makes sense to filter out duplicates.